bedrock/pocket/templates/pocket/includes/platform-footer.html

```diff
@@ -8,14 +8,14 @@
   <nav class="footer-navigation">
     <ul class="footer-navigation-list">
       <li class="footer-navigation-list-item"><a href="https://blog.getpocket.com/">{{ ftl('pocket-footer-blog') }}</a></li>
-      <li class="footer-navigation-list-item"><a href="https://getpocket.com/en/about/">{{ ftl('pocket-footer-about') }}</a></li>
+      <li class="footer-navigation-list-item"><a href="{{ url('pocket.about') }}">{{ ftl('pocket-footer-about') }}</a></li>
       <li class="footer-navigation-list-item"><a href="https://getpocket.com/explore">{{ ftl('pocket-footer-explore') }}</a></li>
       <li class="footer-navigation-list-item"><a href="https://getpocket.com/publisher/">{{ ftl('pocket-footer-publishers') }}</a></li>
       <li class="footer-navigation-list-item"><a href="https://getpocket.com/developer/">{{ ftl('pocket-footer-developers') }}</a></li>
-      <li class="footer-navigation-list-item"><a href="https://getpocket.com/en/tos/">{{ ftl('pocket-footer-terms-of-service') }}</a></li>
-      <li class="footer-navigation-list-item"><a href="https://getpocket.com/en/privacy/">{{ ftl('pocket-footer-privacy-policy') }}</a></li>
+      <li class="footer-navigation-list-item"><a href="/en/tos/">{{ ftl('pocket-footer-terms-of-service') }}</a></li>
+      <li class="footer-navigation-list-item"><a href="/en/privacy/">{{ ftl('pocket-footer-privacy-policy') }}</a></li>
```

Comment:
Suggestion (non-blocking): Any reason not to use the `url()` helper for these two? If not, favour switching to it for better overall consistency (and less dev questioning?) :)

Reply:
Done 😄